apisix/discovery/kubernetes/informer_factory.lua

```diff
@@ -263,6 +263,9 @@ local function list_watch(informer, apiserver)
     local reason, message
     local httpc = http.new()
 
+    informer.continue = ""
+    informer.version = ""
```

Comment:
Is it possible to add some cases to verify it?

Reply:
I've thought about it too, but it's difficult.
Unless we are able to modify the value of "continue" at runtime in some way

Reply:
There is really no simpler way to reproduce. This unit test case is probably not easy to write.